project-jee8-webapp/src/main/java/fr/pantheonsorbonne/ufr27/miage/resource/InfoCentreEndPoint.java

```diff
@@ -46,4 +46,28 @@ public Response sendPeriodicBulletin(TrainWrapper trains) {
 		// }
 	}
 
+	@POST
+	@Path("/nhe")
+	public Response launchPeriodicBulletin() {
```

Comment:
If you want to send stuff to the infogare *periodically*, you'll need to create a thread with an infinite loop to send all the information.
Could replace the previous method?